test/extended/util/framework.go

```diff
@@ -235,7 +236,7 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface,
 			}
 		}
 	}
-	return fmt.Errorf("the deploy did not finish within 3 minutes")
+	return fmt.Errorf("the deploy did not finish within %v minutes", maxWaitingTime)
```

Comment:
+1 on time as a parameter

note I also added the name of the deployment as a parameter; in the case of the jenkins test, there is more than one deployment going on, so it helped to distinguish which one had timed out